fix parameters disordered problem

chongminggao opened the pull request:
1. Fix the parameters displacement.
2. A slightly better style is given.

chongminggao committed:
fix parameters disordered problem

1. Fix the parameters displacement.
2. A slightly better style is given.

pep8speaks commented:
Hello @eagerming! Thanks for updating this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`ptsdae/utils.py`](https://github.com/vlukiyanov/pt-sdae/blob/dee16fe80ac7920dae3c41fecd3fb9cf1ec08b2b/ptsdae/utils.py):

> [Line 53:1](https://github.com/vlukiyanov/pt-sdae/blob/dee16fe80ac7920dae3c41fecd3fb9cf1ec08b2b/ptsdae/utils.py#L53): [W293](https://duckduckgo.com/?q=pep8%20W293) blank line contains whitespace
> [Line 53:1](https://github.com/vlukiyanov/pt-sdae/blob/dee16fe80ac7920dae3c41fecd3fb9cf1ec08b2b/ptsdae/utils.py#L53): [W391](https://duckduckgo.com/?q=pep8%20W391) blank line at end of file



##### Comment last updated at 2019-05-08 03:42:03 UTC

vlukiyanov commented:
Ah cool, thanks. Feel free to make the changes in your comment as it might be more obvious; if possible could you rename `rowInd` to `row_ind` and `colInd` to `col_ind` to keep in line with Python-esque naming style for variables.

chongminggao committed:
make the codes keep in line with python standard

chongminggao committed:
Use the code that is more obvious

And delete the original lines:

reassignment = np.dstack(linear_sum_assignment(count_matrix.max() - count_matrix))[0]
accuracy = count_matrix[reassignment[:, 0], reassignment[:, 1]].sum() / y_predicted.size
return {item[0]: item[1] for item in reassignment}, accuracy
